Unix/doc/setup-ntlm-omi.md

```diff
@@ -0,0 +1,163 @@
+#Enabling SPNEGO/NTLM Authentication on OMI
+
+OMI requests between the client and server have been up to this point
+authenticated via Basic auth. Basic is very simple, but not at all secure, 
+inasmuch as the username and password are transmitted in the clear, with only 
+Base64 encoding which is easly decoded by someone able to see the traffic on the 
+connection. For that reason we only recommend Basic never be used on other than a 
+secure (https) conneciton.
+
+A more secure method of authentication uses Secure Protected Negotitation protocol, which enables
+negotiation of the security protcol from a selection of options, Currently the only option available with
+omi is the* NT Lan manager protocol, version 2 (NTLMV2)* which is an improvment over Basic auth in two ways. 
+
+- The password is hashed using an irreversable algorithm, so the password is more secure than Basic.
+
+- The client and server support encryption of your data over http connections, so SSL certificates are not required. 
+
+In order to use SPNEGO with NTLM, you will need to set up both the client and the server 
+to provide the credentials. 
+
+Note, NTLM support on gss is not available on MacOS. Solaris, Hpux, Aix or older linux distros
+can be modified to support it, but that configuration is not currently supported.  
+
+In the instructions following, the architecture x86_64 is assumed for simplicity. If you are
+using other architectures (for example,  Power8) please make the appropriate substitutions.
+
+###Setting up the machine
+OMI uses the MIt implementation of the Generic Security Services API (libgssapi_krb5.so). This is the 
+version installed by default in virtually all systems using anything like a Unix/linux OS. The MIT GSS 
+allows plugins to be installed to handle different authentication mechanisms. The NTLM protocol plugin is 
+provided by the gss-ntlmssp package. 
+
+Both the client and host machine must be set up to use provide NTLM to the generic security services (gss) library.
+To do this, the gss and gss-ntlmssp packages must be installed and up to date.  The packages required are:
+
+- RHEL 7.3, Centos 7.3 : 
+ 	 krb5-workstation-1.14.1-27.el7.x86_64 or later,   
+  	 gssntlmssp-0.7.0-1.el7.x86_64.
+     
+- Ubuntu 16.04 (xenial): 
+     libgssapi-krb5-2 version  1.13.2+dfsg-5 or later
+	 gss-ntlmssp version 0.7.0 or later.  
+	 Note: The gss-ntlmssp 0.7.0 package is in progress from canonical.  Until it is available, 0.7.0 can be built form sources as with trusty.
+	         
+- Ubuntu 14.04(trusty)
+     libgssapi-krb5-2 version 1.14.3 or later. 
+     gss-ntlmssp 0.7.0 built from sources from  https://fedorahosted.org/gss-ntlmssp/ as gss-ntlmssp is not provided by canonical on trusty.
```

Comment:
Is this still true? Or is it just the patch that made it into the regular Canonical repos?

Also Canonical should be capitalized, it's a company name.